recipe/meta.yaml

```diff
@@ -47,7 +47,6 @@ requirements:
   run:
     - {{ pin_compatible('numpy', lower_bound='1.16') }}
     - boost-cpp
-    - arrow-cpp {{ version }}
```

Comment:
@peaur I tried removing this again because (so far) it wasn't necessary for the CPU builds (see the global diff of this PR). Adding this should at most be necessary for the GPU builds, IIUC?

PS. This is a comment on commit https://github.com/conda-forge/pyarrow-feedstock/pull/101/commits/99fbb55ef702bdf1cea439559c62e44fd7c0f59c, in case it doesn't show up in the PR thread.

Reply:
I ended up readding it after all - there were import errors pointing to a missing `libarrow`, so this is necessary, but I don't understand how it worked before.

Reply:
The `arrow-cpp {{ version }}` is necessary (as you also found out).
The master `arrow-cpp` does not have CUDA enabled (and I guess something else was changed in arrow), so it does not have `arrow-cpp` in `run` section.